.github/workflows/unit-test.yml

```diff
@@ -78,6 +83,9 @@ on:
         type: string
         default: ''
 
+env:
+  UV_SYSTEM_PYTHON: 1
```

Comment:
Env var to avoid it complaining about not using a venv?

Reply:
Yes, uv's default behavior is everything inside venvs, but for GitHub Actions and within standard hosted runner, I don't see a need.